cmake/private/FairRootSummary.cmake

```diff
@@ -0,0 +1,218 @@
+################################################################################
+# Copyright (C) 2014-2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+function(fairroot_summary_header)
+  message(STATUS "  ")
+  message(STATUS "               C O N F I G U R A T I O N   S U M M A R Y")
+  message(STATUS "  ")
+  message(STATUS "  ${Cyan}PROJECT NAME${CR}         ${BGreen}${PROJECT_NAME}${CR}")
```

Comment:
Could you please rename the project to `FairRoot`? Unless something depends on it, some `FAIRROOT_*` variables possibly. If it is too much, then let's do it in a separate PR.

Reply:
I tried. And some variables depend on it. It took me a bit of time to track some of them down. And I still fear I missed some. So let's move that into another PR.